### PR TITLE
Remove all old storage code + highcharts.

### PR DESCRIPTION
**Pull request by jgrund**
Remove all the storage page code in this repo.

In addition, remove highcharts + related code.

Signed-off-by: Joe Grund <joe.grund@intel.com>

### CODE REVIEW

File: static/js/storage_resource_detail.js [hunk 1]
```diff
@@ -1,251 +0,0 @@
-//
```
Comment (petertix) [inline static/js/storage_resource_detail.js]: With the removal of this file, Is the routing for this modified as well?
https://github.com/intel-hpdd/GUI/blob/master/source/iml/app/app-states.js#L39

Comment (jgrund) [inline static/js/storage_resource_detail.js]: No. This repo (old-gui) and GUI are effectively not connected.

Comment (jgrund) [inline static/js/storage_resource_detail.js]: Keep in mind that you are referencing master there as well. That code goes away once the storage patch lands in GUI.

Comment (petertix) [inline static/js/storage_resource_detail.js]: Ok, thanks.

**Pull request merged**